documentation/api/change_log.rst

```diff
@@ -3,6 +3,10 @@
 API change log
 ===============
 
+v2.0 | 2021-04-XX
+"""""""""""""""""
+- [**Breaking change**] Automatic inference of horizons for Post requests re-instantiated, now inferring a belief horizon of zero (leading to the same horizon for each belief).
```

Comment:
Wait, no. Describing this as a breaking change for v2.0 would be wrong for two reasons:
- It should not impact v2.0, but versions <2.0 only. Version 2 already inferred a prior (for non-play servers) or a horizon (for play servers) instead.
- It is not a breaking change, but partially reverting a breaking change by reinstantiating the inference, but changing the inference policy.
I pushed a commit that should fix this issue.

My remaining concern is about structuring the API changelog itself. The entry is listed under v2, but the change only impacts lower versions. I don't have a good answer on how to restructure, but I can explain how I handled this before.

The entries are not listed chronologically, but grouped by the most recent API version they impact, with some earlier entries also impacting multiple versions (which is noted with a comment).

I suggest we move the 3rd breaking change of v2.0-2 to a new entry v1.3-8 (with the same date as v2.0-2 and a comment `*Affects all versions since v1.0*.`), and to move the current entry v2.0-3 to a new entry v1.3-9, also with a comment `*Affects all versions since v1.0*.`

Reply:
"I pushed a commit that should fix this issue." Which one do you mean?

Reply:
Apparently I forgot to actually push [my commit](https://github.com/SeitaBV/flexmeasures/pull/97/commits/c5923ae1243f7f3cbdaf25590c2195014715c1d4). I did just now, after resolving merge conflicts in the changelog.